### PR TITLE
fix: title="null" with disableHtmlSanitization #61

### LINKED ISSUE #61: title="null" with disableHtmlSanitization: true
# Report on twitter

https://twitter.com/kebsworld/status/1652289518249099270
image

# Minimal Reproduction

script:
```
import { render } from "https://deno.land/x/gfm@0.2.1/mod.ts";
const body = "[hello](https://example.com/) World";
const html = render(body, {
  disableHtmlSanitization: true,
});
console.log(html);
```

result:
```
<p><a href="https://example.com/" title="null" rel="noopener noreferrer">hello</a> World</p>
```

But not reproduced with no option:

script: 
```
import { render } from "https://deno.land/x/gfm@0.2.1/mod.ts";
const body = "[hello](https://example.com/) World";
const html = render(body);
console.log(html);
```

result: 
```
<p><a href="https://example.com/" rel="noopener noreferrer">hello</a> World</p>
```

### PR DESCRIPTION
**Pull request by hashrock**
Close #61

### CODE REVIEW

**Review by lino-levan — APPROVED** [review]
LGTM! Nice work.

File: mod.ts [hunk 1]
```diff
@@ -44,15 +44,16 @@ class Renderer extends Marked.Renderer {
     const html = Prism.highlight(code, grammar, language!);
     return `<div class="highlight highlight-source-${language} notranslate"><pre>${html}</pre></div>`;
   }
-
-  link(href: string, title: string, text: string) {
+  link(href: string, title: string | null, text: string) {
     if (href.startsWith("#")) {
       return `<a href="${href}" title="${title}">${text}</a>`;
```
Comment (kt3k) [inline mod.ts:49]: Looks like we also need the same fix here? (for in-page links)

Comment (hashrock) [reply mod.ts]: thanks!

**Review by kt3k — APPROVED** [review]
LGTM. Nice!

**Pull request merged**